lib/cdo/script_config.rb

```diff
@@ -19,20 +20,6 @@ def self.hoc_scripts
   end
 
   def self.csf_scripts
-    %w(
-      20-hour
-      course1
-      course2
-      course3
-      course4
-      coursea-2017
-      courseb-2017
-      coursec-2017
-      coursed-2017
-      coursee-2017
-      coursef-2017
-      express-2017
-      pre-express-2017
-    )
+    ScriptConstants::CATEGORIES.fetch_values(:csf, :csf_2018, :csf_international, :twenty_hour).flatten.uniq
```

Comment:
Should I get all keys in the hash that start with `css` so that this list does not have to be updated every year?